canonical-web-and-design/maas-ui
# refactor(ui): update service to use a slice

## Done

- Replace the service reducers and actions with a slice.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Load the machine list and check that the services were fetched (check the redux state in devtools)

## Fixes

Fixes: canonical-web-and-design/maas-ui#1410
